string SWITCH_TO_ACTION = switch-to-action
string SELECT_ACTION_STEP = select-action-step
string SWITCH_TO_EXPERIMENT = switch-to-experiment
string NEW_EXPERIMENT = new-experiment
string command
uint8 param